lib/signalfx/tracing/instrumentation/pg.rb

```diff
@@ -0,0 +1,36 @@
+module SignalFx
+  module Tracing
+    module Instrumenter
+      module PG
+
+        Register.add_lib :PG, self
+
+        class << self
+
+          def instrument(opts = {})
+            return if @instrumented
+
+            # check for required gems
+            begin
+              require 'pg'
+            rescue LoadError
+              return
+            end
+
+            begin
+              require 'pg/instrumentation'
+            rescue LoadError => e
+              puts e.message
```

Comment:
Not blocking this PR, but did you happen to investigate https://github.com/signalfx/signalfx-ruby-tracing/issues/23?